controller-server/src/test/java/com/yahoo/vespa/hosted/controller/persistence/TenantSerializerTest.java

```diff
@@ -80,7 +81,8 @@ public void cloud_tenant() {
         CloudTenant tenant = new CloudTenant(TenantName.from("elderly-lady"),
                                              Optional.of(new SimplePrincipal("foobar-user")),
                                              ImmutableBiMap.of(publicKey, new SimplePrincipal("joe"),
-                                                               otherPublicKey, new SimplePrincipal("jane")));
+                                                               otherPublicKey, new SimplePrincipal("jane")),
+                                             TenantInfo.EmptyInfo);
```

Comment:
Should set a non-empty `TenantInfo` here, and assert the re-serialized value below to ensure that serialization is correct.

Reply:
I guess. The non empty case is covered below (albeit not in the context of a CloudTenant) - the normal case is that it is empty. I can create another test.

Reply:
Right, I didn't notice the other tests.

Reply:
Fixed. Added one extra test there to cover that the info field is set correctly in the tenant object.